src/clr-angular/forms/checkbox/checkbox.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Directive, ViewContainerRef} from "@angular/core";
+
+import {WrappedFormControl} from "../common/wrapped-form-control";
+
+import {ClrCheckboxContainer} from "./checkbox-container";
+
+@Directive({selector: "[clrCheckbox]"})
+export class ClrCheckboxNext extends WrappedFormControl<ClrCheckboxContainer> {
```

Comment:
This means you agree with the `Next` annotation, which I'm happy with. Speak now or forever hold your peace.

Reply:
🤐